.github/workflows/gh-registry.yml

```diff
@@ -70,3 +70,20 @@ jobs:
         working-directory: 'hello-strongbox-npm'
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  publish-hello-strongbox-maven:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-java@v1
+        with:
+            java-version: 1.11
+      - name: 'Copy Settings'
+        run: cp ~/.m2/settings.xml settings.xml
+      - name: Maven install
+        run: mvn -Dmaven.compiler.s ource=11 -Dmaven.compiler.target=11 install
```

Comment:
There is a typo here. Also, we're currently still using JDK 8. You shouldn't need to set either of these two properties here, but in the [pom.xml](https://github.com/strongbox/strongbox-examples/blob/master/hello-strongbox-maven/pom.xml) by explicitly defining a `<plugins/>` section in the `<build>` and so on.

Reply:
I agree - furthermore passing it as a mvn cli argument is completely unnecessary. The commands should be `mvn clean install` and `mvn deploy`. :)

Reply:
I fixed that, sorry didn't look at the version supported 😄 .